Packs/Core/Integrations/CortexCoreIR/README.md

```diff
@@ -3298,4 +3298,51 @@ Run a shell command on a specific endpoint and return its result.
 | echo | | dummy_id | 2.2.2.2 | name | STATUS_010_CONNECTED | COMPLETED_SUCCESSFULLY |
 | echo hello | hello | dummy_id | 2.2.2.2 | name | STATUS_010_CONNECTED | COMPLETED_SUCCESSFULLY |
 | echo |  | dummy_id2 | 11.11.11.11 | name2 | STATUS_010_CONNECTED | COMPLETED_SUCCESSFULLY |
-| echo hello | hello | dummy_id2 | 11.11.11.11 | name2 | STATUS_010_CONNECTED | COMPLETED_SUCCESSFULLY |
\ No newline at end of file
+| echo hello | hello | dummy_id2 | 11.11.11.11 | name2 | STATUS_010_CONNECTED | COMPLETED_SUCCESSFULLY |
+
+
+### core-add-indicator-rule
+
+***
+Upload IOCs Rule to XSIAM. 
+When `ioc_object` is defined, disregard any other provided arguments as `ioc_object` takes precedence.
+If `vendor_name`, `vendor_reputation`, and `vendor_reliability` are used, only a single vendor is supported. For multiple vendors, utilize an `ioc_object` in JSON format.
+Adding the same indicator with different parameters will result in an update to the existing rule.
```

Comment:
Adding the same indicator, but with different parameters, will update the existing rule.